python/desc/imsim/camera_readout.py

```diff
@@ -448,3 +473,205 @@ def set_phosim_bboxes(amp):
     amp.setRawPrescanBBox(afwGeom.Box2I(afwGeom.Point2I(0, 1),
                                         afwGeom.Extent2I(4, 2000)))
     return amp
+
+PixelParameters = namedtuple('PixelParameters',
+                             ('''dimv dimh ccdax ccday ccdpx ccdpy gap_inx
+                                 gap_iny gap_outx gap_outy preh'''.split()))
+pixel_parameters \
+    = {'E2V': PixelParameters(2002, 512, 4004, 4096, 4197, 4200, 28,
+                              25, 26.5, 25, 10),
+       'ITL': PixelParameters(2000, 509, 4000, 4072, 4198, 4198, 27,
+                              27, 26.0, 26, 3)}
+
+# The image extension headers do not include the CCD vendor so we
+# infer that from the values of the DETSIZE keyword.
+vendors = {'[1:4096,1:4004]': 'E2V',
+           '[1:4072,1:4000]': 'ITL'}
+
+def set_noao_keywords(hdu, slot_name):
```

Comment:
Maybe we can also put in a definition reference for the keywords?  I found this:

http://iraf.noao.edu/projects/ccdmosaic/imagedef/fitsdic.html

Reply:
meh.